joomla/joomla-cms
# Move inline JCaption to caption.js

### Description

This PR moves the inline JCaption js onload call to the external caption.js file.
### How to test
1. Fresh Joomla install with sample data.
2. Install the patch
3. Add an image with caption to a content
4. Check frontend if image caption is there
5. Check the HTML page code. JCaption inline js should no longer be there
### B/C

None.
